.github/CONTRIBUTING.md

```diff
@@ -33,6 +33,8 @@ Run the new binary as: `./bin/glab`
 
 Run tests with: make test or go test ./...
 
+WARNING: Do not run `make test` outside of an isolated environment, it will overwrite your global config.
```

Comment:
I would that you drop this commit and open it in a separate PR... I prefer having PRs focused on its description
```suggestion
```